command/secret.go

```diff
@@ -0,0 +1,30 @@
+package command
+
+import (
+	"github.com/mitchellh/cli"
+	"github.com/pkg/errors"
+)
+
+// Secret command
+type Secret struct {
+	*command
+}
+
+// SecretFactory returns a factory method for the join command
+func SecretFactory() (cli.Command, error) {
+	comm, err := newCommand("nerd secret <subcommand> {opaque, registry}", "set and list secrets", "", nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create command")
+	}
+	cmd := &Project{
```

Comment:
Good catch! That is a mistake indeed, I'll try to write up some documentation about the PR to make your life easier (: